### PR TITLE
Standalone CBMA

### PR DESCRIPTION
**Pull request by pentestiing**
This PR adds a script to easily launch CBMA in standalone mode as well as two related functional tests, one runs in a simulated setup in any device supporting namespaces and the other in real devices :left_luggage:

### CODE REVIEW

File: modules/sc-mesh-secure-deployment/src/2_0/features/cbma/standalone_cbma.py [hunk 1]
```diff
@@ -0,0 +1,193 @@
...
+
+    cleanup_cbma(bat_iface)
+
+    batman(bat_iface)
+
+    for iface in interfaces:
+        try:
+            with open(f"/sys/class/net/{iface}/phy80211/index") as f:
+                index = f.read()
+                wpa_supplicant_control_path = f"/var/run/wpa_supplicant_id{index}/{iface}"
```
Comment (joenpera) [inline modules/sc-mesh-secure-deployment/src/2_0/features/cbma/standalone_cbma.py:115]: Index in wpa_supplicant_id{index} is a different thing than /sys/class/net/{iface}/phy80211/index.   

wpa_supplicant_id{index} is a configuration index which is used in Comm_control based configurations.  Not mapping to phy index.

File: common/tests/functional/test_cbma/cbma_check_standalone.sh [hunk 2]
```diff
@@ -0,0 +1,111 @@
...
+        unset PS1
+
+				cd "$CBMA_DIR"
+
+				python3 standalone_cbma.py &
+
+        pid=$!
+        trap "kill $pid" INT KILL QUIT EXIT
+        wait $pid
+		EOF
```
Comment (joenpera) [inline common/tests/functional/test_cbma/cbma_check_standalone.sh:55]: Wobbly lines :screwdriver:

File: common/tests/functional/test_cbma/cbma_check_standalone.sh [hunk 3]
```diff
@@ -0,0 +1,111 @@
...
+
+    for ip in $IPS; do
+        echo -n "[+] Waiting for all neighbors in CBMA of ${ip}... "
+        command ssh "root@${ip}" sh <<- EOF
+						while batctl meshif bat0 n -H 2>/dev/null | awk -F'[[:space:]]*|s' '\$4>1{NR=0;exit}END{exit(NR=='${bat_neighbors}')}'; do
+								sleep 1
+						done
+				EOF
+        echo "done"
+    done
```
Comment (joenpera) [inline common/tests/functional/test_cbma/cbma_check_standalone.sh:88]: Same as above

**Comment by pentestiing** [comment]
Closing as no longer needed due to the CBMA MVP rewrite :writing_hand:

**Pull request closed without merging**